src/index.html

```diff
@@ -5,9 +5,153 @@
+            <p>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated
+              in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p>
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than
+              200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this
+              area better than other specialists.<br>Working directly with developers - so have more useful information
+              about the profitable deals.
+            </p>
+            <p>
+              <strong>We will never be late!<br>Guaranteed secure transaction - 100%</strong>
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section>
+        <h2 class="heading">Gallery<span class="heading__crown"></span></h2>
+        <div class="gallery">
+          <div class="photos photos-wide-left">
+            <img src="./images/photos/1.jpg" alt="Marina Palms" class="photos__image">
+            <img src="./images/photos/2.jpg" alt="Miami beach" class="photos__image">
+          </div>
+          <p class="gallery__caption">Marina Palms / <a href="#" class="gallery__link">North Miami Beach, FL 33162</a></p>
+          <div class="photos">
+            <img src="./images/photos/3.jpg" alt="USA flag" class="photos__image">
+            <img src="./images/photos/4.jpg" alt="Building" class="photos__image">
+          </div>
+          <p class="gallery__caption">Edition Residens / <a href="#" class="gallery__link">Miami Beach, FL 33139</a></p>
+          <div class="photos photos-wide-right">
+            <img src="./images/photos/5.jpg" alt="Marina Palms" class="photos__image">
+            <img src="./images/photos/6.jpg" alt="Miami beach" class="photos__image">
+          </div>
+          <p class="gallery__caption">Faena House / <a href="#" class="gallery__link">Miami Beach, FL 33140</a></p>
+          <button class="button gallery__button">View all</button>
+        </div>
+      </section>
+
+      <section>
+        <h2 class="heading">Consultation<span class="heading__crown"></span></h2>
+        <div class="consultation">
+          <div class="consultation__text">
+            <h3 class="consultation__heading">How long have you do<br>yourself a gift?</h3>
+            <p>The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his
+              "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found
+              a distant historical.</p>
+          </div>
+          <form class="form consultation__form" method="POST" action="">
+            <label class="form__item"><input type="text" name="name" placeholder="Name" class="input form__input"></label>
```

Comment:
```suggestion
            <label class="form__item">
              <input 
                type="text" 
                name="name" 
                placeholder="Name" 
                class="input form__input"
              >
            </label>
```
улучшаем читаемость, исправь везде